docs/MainSite/Installation.md

```diff
@@ -88,45 +92,61 @@ spack install jemalloc # or from your package manager
 spack install libxsmm
 spack install yaml-cpp@develop
 ```
-
-Next, run `ls spack/share/spack/modules` add one entry of
-```
-module use /path/to/spack/share/spack/modules/YOUR_OS_VERSION_INFO
-```
-for each `YOUR_OS_VERSION_INFO` inside `spack/share/spack/modules` to your
-`~/.bash_profile` or `~.bashrc`.
-
 You can also install CMake, OpenBLAS, Boost, and HDF5 from Spack.
 To load the packages you've installed from Spack run `spack load PACKAGE`,
-or use the `module load` command. Spack allows very flexible configurations and
+or (equivalently) use the `module load` command.
+
+**Note**: Spack allows very flexible configurations and
 it is recommended you read the [documentation](https://spack.readthedocs.io) if
 you require features such as packages installed with different compilers.
 
 ### Building SpECTRE
 
-* Ensure you have all dependencies installed. See
-"Installing Dependencies Using Spack" for one method.
-* Install [Charm++](http://charm.cs.illinois.edu/software)
- for your machine. Further details
- [here](http://charm.cs.illinois.edu/manuals/html/charm++/A.html)
-* Clone [SpECTRE](https://github.com/sxs-collaboration/spectre)
-* Apply the Charm++ patch for your version *after* building Charm++ by running
-`git apply SPECTRE_ROOT/support/Charm/vx.y.z.patch` in the Charm++ directory
-* `mkdir build && cd build`
+After the dependencies have been installed, Charm++ and SpECTRE can be compiled.
+Follow these steps:
+
+1.  Clone [Charm++](http://charm.cs.illinois.edu/software) into `CHARM_DIR`,
+    again a directory of your choice.
+    To use a stable version of Charm++, use `git tag` to see tagged releases,
+    and `git checkout vx.y.z` to switch to a recent release.
+    Follow the directions
+    [here](http://charm.cs.illinois.edu/manuals/html/charm++/A.html) to compile
+    Charm++. The correct target is `charm++` and, for a personal machine, the
+    correct target architecture is likely to be `mpi-linux-x86_64`
```

Comment:
We should also note that we will be providing specific instructions for various HPC systems